app/assets/javascripts/templates/login.html.haml

```diff
@@ -41,4 +41,6 @@
           tabindex: "3",
           type: "submit",
           value: "{{'label_login' | t}}"}
-
+    .row
+      .large-12.columns
+        = ActionController::Base.helpers.link_to "Login with Les Communs", "/user/spree_user/auth/openid_connect", method: :post
```

Comment:
Does `link_to` not work in templates without referencing `Actioncontroller::Base`? That seems like something that could/should be fixed..?

We can use the named route `spree_user_openid_connect_omniauth_authorize_path` instead of passing `"/user/spree_user/auth/openid_connect"` as a string here, right?

Reply:
Yeah I was having trouble with both of those - I've been pushing things up sooner than I normally would so that I can deploy to FR staging to get a more realistic environment to test the auth flow with (the app at Les Communs is configured to redirect to FR staging and not localhost).